corehq/apps/ota/views.py

```diff
@@ -37,6 +38,7 @@
 from .utils import demo_user_restore_response, get_restore_user, is_permitted_to_restore, handle_401_response
 
 
+@location_safe
```

Comment:
this can break fairly easily because it's order dependent (has to be the first decorator). haven't looked into a way to fix, but thought i'd flag